packages/flame_tiled/lib/src/renderable_layers/tile_layer.dart

```diff
@@ -18,15 +18,17 @@ class FlameTileLayer extends RenderableLayer<TileLayer> {
   late List<List<MutableRSTransform?>> indexes;
   final animations = <TileAnimation>[];
   final Map<Tile, TileFrames> animationFrames;
+  final bool _ignoreFlip;
```

Comment:
I think this can be kept public and then the default directly set in the constructor argument